cmd/bnsd/x/blueaccount/handler.go

```diff
@@ -503,3 +509,55 @@ func (h *deleteAccountHandler) validate(ctx weave.Context, db weave.KVStore, tx
 	}
 	return &msg, nil
 }
+
+type deleteAllAccountsHandler struct {
```

Comment:
`deleteAllAccountsHandler` seems like a bit vague name to explain this handler is will delete the accounts registered under a *domain*. Rename the handler or better add a comment.

Reply:
I have [renamed `DeleteAllAccountsMsg` to `FlushDomainMsg](https://github.com/iov-one/weave/pull/1042/commits/e996e1f5d3a51ba3e9c2fcdc62504fbba8be4079).

How about now?